<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equipement</class>
 <widget class="QMainWindow" name="equipement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>627</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Gestion d'equipement</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color:#29ABE2
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>1511</width>
      <height>581</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Link">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Link">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Link">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="view">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>130</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#tableWidget{
background-color:#caeefe;
}</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>90</y>
        <width>315</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>pdf</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>statistique</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>id</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>nom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>type</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>etat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>disponibilite</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>date d'achat</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>460</y>
        <width>295</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="ajoutbutton">
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="search">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>rechercher</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="del_all">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>460</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>supp tout</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>155</y>
        <width>68</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>192</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>192</y>
        <width>154</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>155</y>
        <width>154</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>229</y>
        <width>29</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="type">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>229</y>
        <width>154</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>266</y>
        <width>26</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>303</y>
        <width>82</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>date d'achat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="etat">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>266</y>
        <width>154</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>380</y>
        <width>116</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>image de produit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>340</y>
        <width>83</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>disponibilite</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>303</y>
        <width>100</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>340</y>
        <width>123</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="true_2">
         <property name="text">
          <string>True</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="false_2">
         <property name="text">
          <string>False</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>60</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>calendrier</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>370</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="led">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>449</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modifier">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>121</y>
        <width>68</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>437</x>
        <y>136</y>
        <width>116</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>date d'achat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>438</x>
        <y>204</y>
        <width>83</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>disponibilite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>335</y>
        <width>29</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>228</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>437</x>
        <y>270</y>
        <width>116</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>image de produit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="etatm">
      <property name="geometry">
       <rect>
        <x>567</x>
        <y>336</y>
        <width>154</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="typem">
      <property name="geometry">
       <rect>
        <x>183</x>
        <y>335</y>
        <width>154</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>447</x>
        <y>336</y>
        <width>26</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>etat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomm">
      <property name="geometry">
       <rect>
        <x>183</x>
        <y>228</y>
        <width>154</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="idm_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>183</x>
        <y>121</y>
        <width>154</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>270</y>
        <width>166</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>420</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>420</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QPushButton{
background-color:#99CC99
}</string>
      </property>
      <property name="text">
       <string>confirmer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>68</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="datem">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>140</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>200</y>
        <width>147</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="dispom1">
         <property name="text">
          <string>true</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dispom2">
         <property name="text">
          <string>false</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_stat_equipe">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>831</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>430</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>retourner</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>151</x>
        <y>105</y>
        <width>431</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>470</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>retourner</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_calendar_affichage"/>
       </item>
       <item>
        <widget class="QTableView" name="tableView_calendar"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>251</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #CAEEFE; /* Fond bleu clair pour le groupe */
</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>271</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resource.qrc">:/images/Logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9QQSDQD">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#7EB6D4;
font: 14pt &quot;Tempus Sans ITC&quot;;
border-radius: 25px; 
border: 1.5px solid #29ABE2; 
color:#000000;
QPushButton: {
                     font-size: 14px;
                      transition: all 0.3s ease;
                      }
QPushButton:hover {
                      font-size: 18px;
                      }</string>
     </property>
     <property name="text">
      <string>Equipement</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushbutton_client">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;
border-radius: 25px; 
border: 1.5px solid #29ABE2; 
color:#000000;
QPushButton: {
                     font-size: 14px;
                      transition: all 0.3s ease;
                      }
QPushButton:hover {
                      font-size: 18px;
                      }</string>
     </property>
     <property name="text">
      <string>Client</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_service">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;
border-radius: 25px; 
border: 1.5px solid #29ABE7; 
color:#000000;
QPushButton: {
                     font-size: 14px;
                      transition: all 0.3s ease;
                      }
QPushButton:hover {
                      font-size: 18px;
                      }</string>
     </property>
     <property name="text">
      <string>Service</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_produit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;
border-radius: 25px; 
border: 1.5px solid #29ABE2; 
color:#000000;
QPushButton: {
                     font-size: 14px;
                      transition: all 0.3s ease;
                      }
QPushButton:hover {
                      font-size: 18px;
                      }</string>
     </property>
     <property name="text">
      <string>Produit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_employe">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#99CC99;
font: 14pt &quot;Tempus Sans ITC&quot;;
border-radius: 25px; 
border: 1.5px solid #29ABE2; 
color:#000000;
QPushButton: {
                     font-size: 14px;
                      transition: all 0.3s ease;
                      }
QPushButton:hover {
                      font-size: 18px;
                      }</string>
     </property>
     <property name="text">
      <string>Employe</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1516</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
